Sources/System/Errno.swift

```diff
@@ -1187,6 +1195,19 @@ public struct Errno: RawRepresentable, Error, Hashable, Codable {
   public static var EBADMACHO: Errno { malformedMachO }
 #endif
 
+#if os(Windows)
+  /// Operation canceled.
+  ///
+  /// The scheduled operation was canceled.
+  ///
+  /// The corresponding C error is `ECANCELED`.
+  @_alwaysEmitIntoClient
+  public static var canceled: Errno { Errno(_ECANCELLED) }
+
+  @_alwaysEmitIntoClient
+  @available(*, unavailable, renamed: "canceled")
+  public static var ECANCELED: Errno { canceled }
+#else
```

Comment:
@compnerd If the `internal var _ECANCELLED: CInt { 10103 }` can also be renamed, then this `#if os(Windows)` will exactly match the `#else` below.